controllers/news/getNewsByUser.js

```diff
@@ -0,0 +1,15 @@
+const { News } = require('../../db/models');
+
+async function getNewsByUser(req, res) {
+  try {
+    const news = await News.findAll({
+      include: 'user',
```

Comment:
Do you need `user` info if you're searching for bews by userId? (the same user for all records)